code/modules/admin/view_variables/mass_edit_variables.dm

```diff
@@ -1,265 +1,271 @@
-			to_chat(src, "Finding items...")
-			var/list/items = get_all_of_type(O.type, method)
-			to_chat(src, "Changing [items.len] items...")
-			for(var/thing in items)
-				if (!thing)
-					continue
-				var/datum/D = thing
-				if(many && !new_value)
-					new_value = new type()
-
-				if (D.vv_edit_var(variable, new_value) != FALSE)
-					accepted++
-				else
-					rejected++
-				new_value = null
-				CHECK_TICK
-
-		else
-			to_chat(src, "Finding items...")
-			var/list/items = get_all_of_type(O.type, method)
-			to_chat(src, "Changing [items.len] items...")
-			for(var/thing in items)
-				if (!thing)
-					continue
-				var/datum/D = thing
-				if (D.vv_edit_var(variable, new_value) != FALSE)
-					accepted++
-				else
-					rejected++
-				CHECK_TICK
-
-
-	var/count = rejected+accepted
-	if (!count)
-		to_chat(src, "No objects found")
-		return
-	if (!accepted)
-		to_chat(src, "Every object rejected your edit")
-		return
-	if (rejected)
-		to_chat(src, "[rejected] out of [count] objects rejected your edit")
-
-	log_world("### MassVarEdit by [src]: [O.type] (A/R [accepted]/[rejected]) [variable]=[html_encode("[O.vars[variable]]")]([list2params(value)])")
-	log_admin("[key_name(src)] mass modified [original_name]'s [variable] to [O.vars[variable]] ([accepted] objects modified)")
-	message_admins("[key_name_admin(src)] mass modified [original_name]'s [variable] to [O.vars[variable]] ([accepted] objects modified)")
-
-
-/proc/get_all_of_type(var/T, subtypes = TRUE)
-	var/list/typecache = list()
-	typecache[T] = 1
-	if (subtypes)
-		typecache = typecacheof(typecache)
-	. = list()
-	if (ispath(T, /mob))
-		for(var/mob/thing in GLOB.mob_list)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /obj/machinery/door))
-		for(var/obj/machinery/door/thing in GLOB.airlocks)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /obj/machinery))
-		for(var/obj/machinery/thing in GLOB.machines)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /obj))
-		for(var/obj/thing in world)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /atom/movable))
-		for(var/atom/movable/thing in world)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /turf))
-		for(var/turf/thing in world)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /atom))
-		for(var/atom/thing in world)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /client))
-		for(var/client/thing in GLOB.clients)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else if (ispath(T, /datum))
-		for(var/datum/thing)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
-
-	else
-		for(var/datum/thing in world)
-			if (typecache[thing.type])
-				. += thing
-			CHECK_TICK
+/client/proc/cmd_mass_modify_object_variables(atom/A, var_name)
```

Comment:
did you really just respace this entire file?

Reply:
uhhh lemme check